include/gz/rendering/base/BaseScene.hh

```diff
@@ -809,7 +809,7 @@ namespace gz
       /// \param[in] _name unique object name.
       /// \return Pointer to a text geometry object
       protected: virtual TextPtr CreateTextImpl(unsigned int _id,
-                     const std::string &_name);
+                     const std::string &_name) = 0;
```

Comment:
Did we add this modification?

Reply:
Yes, in [this PR](https://github.com/DeepX-inc/gz-rendering/pull/10/files#diff-20f200b860d03f923ff5c6e3c2badb4cc20d29e9993b00d79a909b5c8b7d8c28).
There used to be `= 0` in ancient times, but for some reasons it was dropped. Kristjan added it again in the above PR.